demo/page.md

```diff
@@ -4,6 +4,7 @@ permalink: /docs/
 
 layout: post
 sidenav: docs
+sticky_sidenav: true
```

Comment:
I'm wondering if this sticky side nav option should be at the `config.yml` level to be applied site wide?

Reply:
Possibly, I think it’s good to allow it as optional per-page though

Reply:
I don't feel super strongly about it per page is fine by me and you could always use default values to set it for all pages

Reply:
Remove the `sticky_sidenav` line and we should be good to go! 🎉